.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
```

Comment:
could  you update git-version  to the dotnet tool variant too?  the current one we use doesn't work on  Unix